### PR TITLE
Tune up RTC module to support fileset events

### PR DESCRIPTION
**Pull request by KutsenkoA**

### CODE REVIEW

File: src/api/realtime/realTimeModule.ts [hunk 1]
```diff
@@ -1,13 +1,22 @@
 import { ReflectiveInjector } from "injection-js";
+import { IResource } from "jexia-sdk-js/api/core/resource";
```
Comment (marlosin) [inline src/api/realtime/realTimeModule.ts]: relative path?

File: src/api/dataops/dataset.ts [hunk 2]
```diff
@@ -68,6 +68,13 @@ export class Dataset<T extends object = any, D extends DatasetInterface<T> = Dat
     return this.datasetName;
   }
 
+  /**
+   * Resource type of the dataset
+   */
+  public get resourceType(): ResourceType {
```
[the remark this answers is not in the document]
Comment (KutsenkoA) [reply src/api/dataops/dataset.ts]: agree, fixed :+1:

File: src/api/realtime/realTimeModule.ts (hunk 1 above)
Comment (KutsenkoA) [reply src/api/realtime/realTimeModule.ts]: yeah, IDE put this because of `path` in `typescript.json`. Wishing to get rid of this :)

Comment (marlosin) [reply src/api/realtime/realTimeModule.ts]: I knew it :laughing:

**Pull request merged**